docs/dotnet-five.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What is .NET 5
+description: In this article you will learn about .NET 5, a cross platform, and open-source project that convergences .NET framework and .NET Core together.
+ms.date: 08/31/2020
+ms.topic: overview
+ms.author: dapine
+author: IEvangelist
+---
+
+# What is .NET 5
+
+The .NET ecosystem is being simplified with the convergence of .NET Core and .NET Framework. Naming things is hard, but unifying versions of two disparate .NET implementations can be harder. This article details .NET 5, which can be thought of as .NET Core vNext. With .NET Core concluding at version 3.1 and .NET Framework at version 4.8, to avoid ambiguity the next logical version is .NET version 5.0.
```

Comment:
>  The .NET ecosystem is being simplified with the convergence of .NET Core and .NET Framework.

As mentioned in the description, this isn't quite accurate. It's really the next evolution of .NET Core, but we're dropping the "Core" to strongly indicate that this is the future of .NET moving forward. The rest of this paragraph is good though.